src/form/Checkbox/Checkbox.story.tsx

```diff
@@ -11,6 +11,30 @@ export const Simple = () => {
   return <Checkbox checked={state} label="Check me" onChange={setState} />;
 };
 
+export const WithoutLabel = () => {
+  const [state, setState] = useState(true);
+  return <Checkbox checked={state} onChange={setState} />;
+};
+
+export const LabelPosition = () => {
```

Comment:
In the future, let's break out feature changes from styling changes for easier merging.

Reply:
it makes sense but is not always possible to break